Library/Formula/simavr.rb

```diff
@@ -0,0 +1,24 @@
+class Simavr < Formula
+  desc "a lean, mean and hackable AVR simulator for linux & OSX "
+  homepage "https://github.com/buserror/simavr"
+  url "https://github.com/buserror/simavr/archive/v1.2.tar.gz"
+  sha256 "c7f645b323d74247409b50ea6915f1f324a7ee2a799ef107613626801defc070"
+
+  depends_on "osx-cross/avr/avr-gcc"
```

Comment:
You might well find that an easier path, or hosting this in a tap yourself, aye. The latter is easier than it probably sounds; tap creation and maintenance is a fairly pleasant process. There's some documentation [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) if you're interested in that option.